app/developers/templates/developers/collection_submit_confirmation.html

```diff
@@ -0,0 +1,38 @@
+{% from "govuk_frontend_jinja/components/panel/macro.html" import govukPanel %}
+{% extends "developers/access_grant_funding_base.html" %}
+
+{% block pageTitle %}
+  Collection submitted {{ collection_helper.name }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% set reference = collection_helper.id | string | truncate(8, True, "") %}
+      {% set reference_html %}
+        Your reference number
+        <br />
+        <strong>{{ reference }}</strong>
+      {% endset %}
+      {{
+        govukPanel({
+          "titleText": "Collection submitted",
+          "html": reference_html
+        })
+      }}
+
+      {# <p class="govuk-body">We have sent you a confirmation email.</p> #}
+      <p class="govuk-body">We do not currently send a confirmation email.</p>
```

Comment:
I think we'll definitely send this - do we want to bootstrap a template and ping it over for content?